core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -168,21 +170,24 @@ class EMConfig {
             }
 
             var enumValues = ""
-
+            var experimentalValues =""
+            var validValues = ""
             val returnType = m.returnType.javaType as Class<*>
-
             if (returnType.isEnum) {
                 val elements = returnType.getDeclaredMethod("values")
                         .invoke(null) as Array<*>
-
                 enumValues = elements.joinToString(", ")
+               val experimentElements= elements.filter{ it is WithExperimentalOptions && it.isExperimental()}
+                val validElements= elements.filter{ it is WithExperimentalOptions && !it.isExperimental()}
```

Comment:
the first check is fine, but this latter one is not, as it is not the negation of the first.
ie, if a parameter is not marked with  `WithExperimentalOptions` then all of its enum values are implicitely not experimental, ie:
 `it !is WithExperimentalOptions || !it.isExperimental()`

Furthermore, make sure to run the unit tests before committing, eg `ConfigToMarkdownTest`.
For example, here where is `experimentalValues ` in generating the output? ie the code that does output `Experimental values`

Also, if `experimentalValues.isBlank()`, then  `Experimental values` should not be outputted